spec/models/item_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Item, :type => :model do
+  let (:order) { Order.create(users_id: 1, status:"ordered", total: 1700)}
+  let (:item) { Item.create(title: "Green Chili Burrito", description:"Amazingly spicey burrito", price:655, image: "image/path")}
```

Comment:
Line is too long. [130/80]<br>Space missing after colon.<br>Space missing inside }.